frontend-demo/src/services/openedx/client.ts

```diff
@@ -42,7 +42,7 @@ const fetchUserMe = (): Promise<UserMeResponse> => {
     throw new Error("NEXT_PUBLIC_OPENEDX_API_BASE_URL is not defined")
   }
 
-  const url = `${baseUrl}api/user/v1/me/`
+  const url = `${baseUrl}api/user/v1/me`
```

Comment:
In RC, this hits https://courses-qa.mitxonline.mit.edu/api/user/v1/me/, which does not work.

https://courses-qa.mitxonline.mit.edu/api/user/v1/me does work (Try it).